docs/snapshotting/snapshot-support.md

```diff
@@ -661,11 +661,5 @@ m6i won't work on an m6g instance.
 
 | *CPU family* | *taken on host kernel* | *restored on host kernel* | *working?* |
 | ------------ | ---------------------- | ------------------------- | ---------- |
-| **all**      | 5.10                   | 6.1                       | Y          |
-| **all**      | 6.1                    | 5.10                      | Y          |
```

Comment:
just double checking that the change to `N` here is intentional? Iirc it is, but just verifying

Reply:
Yeah the previous one was mistaken.

Reply:
That's what I thought, thanks for confirming :)